lighthouse-plugin-ad-speed-insights/utils/graph.js

```diff
@@ -14,8 +14,10 @@
 
 // @ts-ignore
 // eslint-disable-next-line
-const BaseNode = require('lighthouse/lighthouse-core/lib/dependency-graph/base-node');
+const BaseNode = require("lighthouse/lighthouse-core/lib/dependency-graph/base-node");
```

Comment:
What does this accomplish?

Reply:
I was trying an auto-formatter and accidentally made this change. Reverted